docs/editors/vscode.md

```diff
@@ -177,33 +177,32 @@ guide them. In the end users should end up with something like this:
       "type": "scala",
       "request": "launch",
       // configuration name visible for the user
-      "name": "Main class",
+      "name": "Launch Main",
       // full name of the class to run
       "mainClass": "com.example.Main",
       // optional arguments for the main class
       "args": [],
       // optional jvm properties to use
-      "jvmOptions": [],
-      // optional build target name in case there more than one
-      // class with the same name
-      "buildTarget": "root"
+      "jvmOptions": []
     },
     // Test class configuration
     {
       "type": "scala",
       "request": "launch",
       // configuration name visible for the user
-      "name": "Test class",
+      "name": "Launch Test",
       // full name of the class to run
-      "testClass": "com.example.Test",
-      // optional build target name in case there more than one
-      // class with the same name
-      "buildTarget": "root"
+      "testClass": "com.example.Test"
     }
   ]
 }
 ```
 
+You can add an optional build target name in case there more than one class with the same name. Inside `"configurations":` add the key `buildTarget` with your target name, e.g. `root`:
```

Comment:
```suggestion
You can also add an optional build target name, which is needed in case there are more than one class with the same name or when launching a class from outside the project. Inside `"configurations":` add the key `buildTarget` with your target name, e.g. `root`:
```

We can also add a paragraph explaining what a build target is after the json:

```
Build target name corresponds to your project name. For example in sbt for `lazy val interfaces = project` the name of the build target will be `interfaces` for sources and `interfaces-test` for tests. To make sure what are the target names please run the command `Metals: Run Doctor`.
```